lib/llm/providers/gemini.rb

```diff
@@ -13,8 +13,8 @@ class Gemini < Provider
 
     ##
     # @param secret (see LLM::Provider#initialize)
-    def initialize(secret)
-      super(secret, HOST)
+    def initialize(secret, options = {})
+      super(secret, host: HOST, **options)
```

Comment:
Why is this provider different to the others ?

Reply:
Fixed